bps/templates/bps_cti.yaml

```diff
@@ -15,5 +15,5 @@ payload:
   payloadName: "${INSTRUMENT}_${TICKET}_cti"
   output: "${USER_CALIB_PREFIX}${INSTRUMENT}/calib/${TICKET}/${TAG}/ctiGen.${RERUN}"
   butlerConfig: "${REPO}"
-  inCollection: "${USER_CALIB_PREFIX}${INSTRUMENT}/calib/${TICKET},${RAW_COLLECTION},${CALIB_COLLECTIONS}"
+  inCollection: "${LINEARIZER_GEN},${USER_CALIB_PREFIX}${INSTRUMENT}/calib/${TICKET},${RAW_COLLECTION},${CALIB_COLLECTIONS}"
```

Comment:
Is this the correct order; or should the `calib/TICKET` come first (it does everywhere else).  Also ... is `LINEARIZER_GEN` guaranteed to exist as an env var?
The problem is that we need the `linearizerPtc` I guess.  In which case does that need to be a certified calib ... I fear this is fragile.